pkg/controllers/awsloadbalancercontroller/credentials_request.go

```diff
@@ -0,0 +1,181 @@
+package awsloadbalancercontroller
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	albo "github.com/openshift/aws-load-balancer-operator/api/v1alpha1"
```

Comment:
Can you group the imports are described earlier.

Reply:
My bad, forgot about the other files. Fixed across the files.